module/VuFindSearch/src/VuFindSearch/Backend/SRU/Connector.php

```diff
@@ -221,7 +221,7 @@ protected function call($method = 'GET', $params = null, $process = true)
             $queryString = implode('&', $query);
         }
 
-        $this->debug('Connect: ' . print_r($this->host . '?' . $queryString, true));
+        $this->debug('Connect: ' . $this->varDump($this->host . '?' . $queryString));
```

Comment:
This looks like a plain string; should we omit the varDump entirely?

Reply:
Yes, there is no need for dumping a string, fixed